forumhelpers/index.html

```diff
@@ -43,20 +43,25 @@ <h3 class="forumHelpersHeader">Below is a list of current Forum Helpers, along w
 
 		<div class="searchBlock">
 			<h3>Search for a Forum Helper</h3>
-			<input type="text" id="searchBar" placeholder="Username" oninput="checkUser()">
+			<input type="text" id="searchBar" placeholder="Username" oninput="checkUser(), updateUserChoice()" onkeydown="if (event.keyCode == 13); document.getElementById('searchButton').click()">
 			<button id="searchButton" onclick="searchUser()" disabled title="Invalid Username">Search</button>
+			<div id="searchOptions"></div>
+			<br>
```

Comment:
Could you add another `<br>` tag right under this one (or make the line height x2 bigger) since search bar and the checkbox look a bit close together.

(Also, not sure what others feel about this but I kind of feel like there shouldn't be a checkbox and it should just hide matching results by default since I'm not sure why someone would want to see the whole list and just a dropdown when searching that has the possible results, but that is for another issue and isn't apart of this changes requested thing)